src/Surfnet/StepupMiddleware/ApiBundle/Configuration/Service/ConfiguredInstitutionService.php

```diff
@@ -40,4 +42,20 @@ public function getAll()
     {
         return $this->repository->findAll();
     }
+
+
+    /**
+     * @return InstitutionListing[]
+     */
+    public function getAllAsInstitution()
+    {
+        $configuredInstitutions = $this->repository->findAll();
+
+        $result = [];
+        foreach ($configuredInstitutions as $institution) {
+            $result[] = InstitutionListing::createFrom(new Institution($institution->institution->getInstitution()));
```

Comment:
It was done in order not make changes in the StepupMiddlewareClient. This because the json serialization is handled differently for an `Institution` and an `InstitutionListing`. I agree it's better to get it fixe. Shall we discuss IRL what we want to do?